pandas/core/dtypes/missing.py

```diff
@@ -1,28 +1,20 @@
 """
-missing types & inference
-"""
```

Comment:
why did this change?

Reply:
Accidental, will fix. When I had errors, I copy pasted imports from master and reran isort to attempt to fix the errors and the spacing on some of the comments messed up.